### ISSUE TITLE
Some icons of mouse-over information are not drawn on 1.0X scale

### ISSUE THREAD

**Issue by HiromuHota** [HiromuHota/pentaho-kettle#28]
![zoom](https://cloud.githubusercontent.com/assets/20668349/22261672/ab1e6bec-e222-11e6-9f21-bc18dc2a2fb7.png)

**Comment by HiromuHota** [comment]
The mouse-over information is always drawn at 1.0X scale no matter what magnification is chosen for the rest.
The root cause of this problem is that draw operations are not always executed in order.
The draw operations are roughly as below:
```
// Scale back to 1.0X
gc.setTransform( translationX, translationY, 0, BasePainter.FACTOR_1_TO_1 );
// Draw mini icons
for ( int i = 0; i < miniIcons.length; i++ ) {
  EImage miniIcon = miniIcons[i];
  gc.drawImage( miniIcon, xIcon, yIcon, BasePainter.FACTOR_1_TO_1 );
}
// Scale to magnification
gc.setTransform( translationX, translationY, 0, magnification );
```

However, as reported by [here](https://stackoverflow.com/questions/17416706/canvas-image-drawing-order) and there, the drawImage is not always executed in order.
`drawImage` is delayed until the image is downloaded, while other operations like `setTransform` can be executed instantly.
As a result, what happened was some of `drawImage` were executed after scaling back to chosen magnification (150% in the image above).

**Comment by HiromuHota** [comment]
Resolved by c1b79c8dbf02c6263130f4d332a6bc362d6202a6.
